### PR TITLE
detect network - error messages - UX fixes

### PR DESCRIPTION
**Pull request by codaelux**
- [x] added error toast for errors

- [x] added info toast for info display

- [x] added detect network 

- [x] this PR is responsive

- [x] session shows only when a new session or join session is valid 

- [x] optimised form display for create session

Screenshot 2020-09-27 at 03 49 11

Screenshot 2020-09-27 at 03 51 13

Screenshot 2020-09-27 at 03 48 38

### CODE REVIEW

**Comment by eltNEG** [comment]
I like the changes you made. One thing I think we should consider reversing is the background of the session ID that was changes to blue. The new changes introduced things that feels off - A blue background that competes with the call to action button, a margin that makes the form looks off and it doesn't hides the copy button and session text on mobile view.

## OLD
Screenshot 2020-09-27 at 8 13 52 AM

## NEW
Screenshot 2020-09-27 at 8 15 55 AM

File: components/Intro/index.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+
+
+export default function Intro(){
```
Comment (eltNEG) [inline components/Intro/index.tsx:4]: I can't find where this is being used. We should remove it if it is not used anywhere.

File: components/Intro/index.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    // const {baseURL} = useBaseURL()
+    return (
+        <section className={`w-full bg-hero-bg bg-opacity-90 p-10 bg-no-repeat bg-cover bg-center h-hero-small tablets:bg-cover tablets:bg-center tablets:h-hero-big`}>
+            <div className='pt-8 bg-right-bottom text-white w-hero-txt h-auto tablets:w-hero-big tablets:px-20 md:w-3/6 md:pt-24'>
+                <h1 className="font-bold text-3xl leading-tight mb-3 tablets:text-5xl md:mb-5 lg:text-6xl">Secured Excrow Service</h1>
+                {/* <p className="text-2xl tablets:text-3xl lg:text-2rem ">{heroText}</p>
+                <div className="mt-4 flex justify-between tablets:mt-10 tablets:pr-10">
+                    <Button text="How it works" Icon={WorksIcon} />
+                    <Button text="Join on twitter" Icon={TwitterIcon} />
+                </div> */}
```
Comment (eltNEG) [inline components/Intro/index.tsx:14]: We should consider removing the dead codes

**Review by eltNEG — CHANGES_REQUESTED** [review]

**Comment by codaelux** [comment]
> I like the changes you made. One thing I think we should consider reversing is the background of the session ID that was changed to blue. The new changes introduced things that feel off - A blue background that competes with the call to action button, a margin that makes the form looks off, a square border, and it doesn't hide the copy button and session text on mobile view.
> 
> ## OLD
> Screenshot 2020-09-27 at 8 13 52 AM
> 
> ## NEW
> Screenshot 2020-09-27 at 8 15 55 AM

Nothing is off here .... its an information that should be displayed to a user its meant to highlight a bg to show that

**Pull request merged**